daemon/daemon.go

```diff
@@ -231,6 +236,8 @@ func (c *Command) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			maintTransmitter.transmitMaintenance(errorKindSystemRestart, "system is restarting")
 		case state.RestartDaemon:
 			maintTransmitter.transmitMaintenance(errorKindDaemonRestart, "daemon is restarting")
+		case state.StopDaemon:
+			maintTransmitter.transmitMaintenance(errorKindDaemonRestart, "daemon is stopping")
```

Comment:
"daemon is stopping to wait for socket activation".. not sure if it's too long?

Reply:
I think thats fine